control-plane/config/crd/patches/webhook_in_httprouteretryfilters.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
I think that some of these files are probably not necessary? Do we have the equivalent generated elsewhere or are we missing/including too many kubebuilder tags?

Reply:
I believe they were generated as part of the initial boilerplate generating command. I just went ahead and cleaned them up since they don't appear to be tied to our kubebuilder tags.